app/src/main/java/com/example/android_challenge_02/QuoteList.kt

```diff
@@ -4,12 +4,17 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import androidx.activity.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.example.android_challenge_02.databinding.ActivityQuoteListBinding
 
 class QuoteList : AppCompatActivity() {
 
     private lateinit var binding: ActivityQuoteListBinding
 
+    private var layoutManager: RecyclerView.LayoutManager? = null
```

Comment:
Você não precisa colocar isso aqui , na hora de utilizar vc pode referencia-la diretamente da seguinte forma na linha 35 do codigo:
`binding.recyclerView.layoutManager = LinearLayoutManager(this)`